plugins/inputs/github/github.go

```diff
@@ -46,6 +47,9 @@ const sampleConfig = `
 
   ## Timeout for HTTP requests.
   # http_timeout = "5s"
+
+	## Enable collection of additional fields (e.g. open/closed pull requests)
+	# additional_fields = true
```

Comment:
Spaces, not tabs. 

Moreover it means that by default it should be `true` and it is `false` (default zero value for boolean)